Exiled.API/Features/Core/EObject.cs

```diff
@@ -540,14 +540,17 @@ public static T FindActiveObjectOfType<T>(Func<EObject, bool> predicate)
         public static T[] FindActiveObjectsOfType<T>(Func<EObject, bool> predicate)
             where T : EObject
         {
-            List<T> objects = new();
+            List<T> objects = ListPool<T>.Pool.Get();
             foreach (EObject @object in InternalObjects.Where(predicate))
             {
                 if (@object.Cast(out T obj))
                     objects.Add(obj);
             }
 
-            return objects.ToArray();
+            T[] array = objects.ToArray();
+            ListPool<T>.Pool.Return(objects);
```

Comment:
Should we add a `ListPool<T>.Pool.ToArrayReturn(...`)?

Reply:
Good idea, will add later today